lua/sui_scoreboard/client/scoreboard.lua

```diff
@@ -86,13 +87,51 @@ function PANEL:Init()
 	self.lblTeam:SetText( "Rank" )
 	
 	self.lblMute = vgui.Create( "DImageButton", self)
+	
+	self.connectingPlayers = {}
+	
+	net.Receive("SUIScoreboardPlayerConnecting", function()
+		local id = net.ReadInt(32)
+		local name = net.ReadString()
+		local steamid = net.ReadString()
+		local steamid64 = net.ReadString()
+		
+		self.connectingPlayers[id] = {name, steamid, steamid64}
+		--print("player " .. name .. " connected, adding to list. List now has " .. #self.connectingPlayers .. " stuff")
+	end )
+	
+	gameevent.Listen( "player_disconnect" )
+	hook.Add( "player_disconnect", "suiscoreboardPlayerDisconnect", function( data )
+		local name = data.name			// Same as Player:Nick()
```

Comment:
I think you should replace C comments with Lua ones
```suggestion
		local name = data.name			-- Same as Player:Nick()
```